### PR TITLE
Add a standalone test fixture for distributed tests

### PR DESCRIPTION
**Pull request by ankitgola005**
<details>
  <summary><b>Add a standalone test fixture for distributed tests</b></summary>

- [ ] Was this discussed/agreed via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## What does this PR do?

Fixes # (issue).

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in GitHub issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/test_pytorch/test_accelerator.py [hunk 1]
```diff
@@ -11,31 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import copy
```
Comment (jerome-habana) [inline tests/test_pytorch/test_accelerator.py:14]: try without the split too

**Review by jerome-habana — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changed Multicard tests to run standalone ([#118](https://github.com/Lightning-AI/lightning-Habana/pull/118))
```
Comment (Borda) [inline CHANGELOG.md:17]: Testing and docs change doesn't need to be listed in the chlog

File: tests/test_pytorch/test_accelerator.py [hunk 3]
```diff
@@ -399,7 +400,8 @@ def test_hpu_parallel_reduce_op_strategy_default():
     assert strategy.reduce_op == "sum"
 
 
-@pytest.mark.skip(reason="TBD : Fix pytest issues")
+@pytest.mark.standalone()
+@pytest.mark.skip(reason="fix ddp pytest issue")
```
Comment (Borda) [inline tests/test_pytorch/test_accelerator.py:404]: This is still skipped...

File: tests/test_pytorch/test_accelerator.py [hunk 4]
```diff
@@ -460,7 +462,8 @@ def test_reduce_op_strategy(tmpdir, hpus, reduce_op, expectation):
         trainer.fit(_model)
 
 
-@pytest.mark.skip(reason="TBD : Fix pytest issues")
+@pytest.mark.standalone()
+@pytest.mark.skip(reason="fix ddp pytest issue")
```
Comment (Borda) [inline tests/test_pytorch/test_accelerator.py:466]: This one too

File: .azure/hpu-tests.yml [hunk 5]
```diff
@@ -104,40 +104,38 @@ jobs:
...
           test_pytorch/test_datamodule.py \
           test_pytorch/test_profiler.py \
-          --forked --hpus 1 --junitxml=hpu1_test-results.xml
+          --hpus 1 --junitxml=hpu_test-torch-results.xml
       workingDirectory: tests/
-      displayName: 'Single card test'
+      displayName: 'HPU General tests'
 
     - bash: |
         set -ex
```
Comment (Borda) [inline .azure/hpu-tests.yml:119]: With single command this is not needed

File: .azure/hpu-tests.yml [hunk 6]
```diff
@@ -104,40 +104,38 @@ jobs:
...
           FP32_LIST=test_pytorch/ops_bf16.txt \
           python -m pytest -sv test_pytorch/test_precision.py \
           -k test_autocast_operators_override --runxfail \
-          --forked --junitxml=hpu_precision_test_override-results.xml
+          --junitxml=hpu_precision_test_override-results.xml
       workingDirectory: tests/
       displayName: 'HPU precision test'
 
+    - bash: |
+        set -ex
```
Comment (Borda) [inline .azure/hpu-tests.yml:135]: The same here, it make sense only with multiple commands

File: .azure/hpu-tests.yml (hunk 5 above)
Comment (jerome-habana) [reply .azure/hpu-tests.yml:119]: taken care in #119

File: .azure/hpu-tests.yml (hunk 6 above)
Comment (jerome-habana) [reply .azure/hpu-tests.yml:135]: taken care in #119

File: CHANGELOG.md (hunk 2 above)
Comment (jerome-habana) [reply CHANGELOG.md:17]: taken care in #119

File: tests/test_pytorch/test_accelerator.py (hunk 3 above)
Comment (jerome-habana) [reply tests/test_pytorch/test_accelerator.py:404]: parameterized dist test have issues. We still need a mechanism to run them in separate sessions. As of now enabling some tests in  #119

File: tests/test_pytorch/test_accelerator.py (hunk 4 above)
Comment (jerome-habana) [reply tests/test_pytorch/test_accelerator.py:466]: same as above

**Pull request merged**